README.md

```diff
@@ -467,6 +492,37 @@ const link = new WebSocketLink({
 
 </details>
 
+<details id="apollo-studio-explorer">
+<summary><a href="#apollo-studio-explorer">🔗</a> Client usage with [Apollo Studio Explorer](https://www.apollographql.com/docs/studio/explorer/additional-features/#subscription-support)</summary>
+
+In Explorer Settings, click "Edit" for "Connection Settings" and select `graphql-ws` under "Implementation".
+
+</details>
+
+<details id="graphiql">
+<summary><a href="#graphiql">🔗</a> Client usage with [GraphiQL](https://github.com/graphql/graphiql)</summary>
+
+```typescript
+import * as React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Let's not mix default import styles.
```suggestion
import React from 'react';
import ReactDOM from 'react-dom';
```

Reply:
Sure, I was just copying from GraphiQL's docs (I don't use GraphiQL).